File: app/src/main/java/com/parabola/newtone/mvp/view/CreatePlaylistView.java
package com.parabola.newtone.mvp.view;

import moxy.MvpView;
import moxy.viewstate.strategy.alias.OneExecution;

@OneExecution
public interface CreatePlaylistView extends MvpView {

    void focusOnInputField();

    void showPlaylistTitleIsEmptyError();
    void showPlaylistTitleAlreadyExistsError();

    void showPlaylistCreatedToast(String playlistTitle);

    void closeScreen();
}

File: app/src/main/java/com/parabola/newtone/mvp/view/CreatePlaylistView.kt
package com.parabola.newtone.mvp.view

import moxy.MvpView
import moxy.viewstate.strategy.alias.OneExecution

@OneExecution
interface CreatePlaylistView : MvpView {
    fun focusOnInputField()

    fun showPlaylistTitleIsEmptyError()
    fun showPlaylistTitleAlreadyExistsError()

    fun closeScreen()
}
